addons/mass_mailing/wizard/mailing_contact_import.py

```diff
@@ -16,7 +16,7 @@ def action_import(self):
         """Import each lines of "contact_list" as a new contact."""
         self.ensure_one()
 
-        contacts = tools.email_split_tuples(self.contact_list)
+        contacts = tools.email_split_tuples(', '.join(self.contact_list.splitlines()))
```

Comment:
@pyduf Can you write a quick unit test for that one?

Reply:
Yes good idea. Actually there was already one. I have slightly modified it to introduce invalid line not only at the end so it covers the case we're fixing (added also a comma in an invalid line and a blank line).